app/experimenter/nimbus-ui/src/components/PageEditBranches/FormBranches/index.tsx

```diff
@@ -248,6 +257,23 @@ export const FormBranches = ({
           )}
         </Form.Group>
 
+        <Form.Row>
+          <Form.Group as={Col} controlId="isRollout">
+            <Form.Check
+              data-testid="is-rollout-checkbox"
+              onChange={handleIsRollout}
+              checked={!!isRollout}
+              type="checkbox"
+              label="This is a rollout (single branch)"
+            />
+            {fieldMessages?.is_rollout && (
+              <Alert data-testid="sticky-required-warning" variant="danger">
+                {fieldMessages?.is_rollout}
+              </Alert>
+            )}
```

Comment:
Would it be better to use the [Form.Check props](https://react-bootstrap.github.io/forms/checks-radios/#form-check-props) `isInvalid` and `feedback` instead of the Alert?

```typescript
isInvalid={!!(fieldMessages?.is_rollout)}
feedback={fieldMessages?.is_rollout}
```

Reply:
good point thanks @mikewilli I have updated as per suggestion :)